sdk/storage/azure-storage-file/azure/storage/file/file_client.py

```diff
@@ -784,10 +785,6 @@ def resize_file(self, size, timeout=None, **kwargs): # type: ignore
     def list_handles(self, marker=None, timeout=None, **kwargs):
```

Comment:
We need to remove the `marker` argument.

Reply:
done